aspnetcore/fundamentals/app-state.md

```diff
@@ -1,236 +1,191 @@
 ---
-title: Managing Application State | Microsoft Docs
+title: Session and application state | Microsoft Docs
 author: rick-anderson
-description: 
-keywords: ASP.NET Core,
+description: Approaches to preserving application and user (session) state between requests.
+keywords: ASP.NET Core, Application state, session state, querystring, post
 ms.author: riande
 manager: wpickett
-ms.date: 10/14/2016
+ms.date: 1/14/2017
 ms.topic: article
 ms.assetid: 18cda488-0769-4cb9-82f6-4c6685f2045d
 ms.technology: aspnet
 ms.prod: aspnet-core
 uid: fundamentals/app-state
 ---
-# Managing Application State
+<!-- Review questions
 
->[!WARNING]
-> This page documents version 1.0.0-rc1 and has not yet been updated for version 1.0.0
+When does session clear the backing store? When the session expires? After you close the browser and the session cookie is deleted - what happens to your session data?
 
-By [Steve Smith](http://ardalis.com)
+`IdleTimeout` is  independent of the cookie's expiration.
+Is there a way to set the session ID cookie timeout? What happens to the session cookie after the session is expired?
+-->
 
-In ASP.NET Core, application state can be managed in a variety of ways, depending on when and how the state is to be retrieved. This article provides a brief overview of several options, and focuses on installing and configuring Session state support in ASP.NET Core applications.
+# Session and application state
 
-[View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/fundamentals/app-state/sample)
+By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Steve Smith](http://ardalis.com)
 
-## Application State Options
+HTTP is a stateless protocol; the Web server treats each HTTP request as an independent request. The server retains no knowledge of variable values that were used in previous requests. This article discusses various approaches to preserving application and user (session) state between requests. Although preserving state is often derided, judicious use of state can reduce app complexity and increase performance.
```

Comment:
derided? by REST folks, yes. Don't mention it unless you're going to include the arguments against state.